### PR TITLE
FastAPI OTEL instrumentation

### PR DESCRIPTION
**Pull request by khoaguin**
Closes https://github.com/OpenMined/syft-internal-issues/issues/233

### CODE REVIEW

**Comment by khoaguin** [comment]
image

**Comment by khoaguin** [comment]
closes https://github.com/OpenMined/syft-internal-issues/issues/233

File: syftbox/server/server.py [hunk 1]
```diff
@@ -166,6 +171,7 @@ async def lifespan(app: FastAPI, settings: Optional[ServerSettings] = None):
 
 
 app = FastAPI(lifespan=lifespan)
+FastAPIInstrumentor.instrument_app(app=app, http_capture_headers_server_request=["email"])
```
Comment (yashgorana) [inline syftbox/server/server.py:174]: where is this email coming from?

Comment (yashgorana) [reply syftbox/server/server.py]: @khoaguin please check the changes i made

Comment (khoaguin) [reply syftbox/server/server.py]: > where is this email coming from?

image

**Review by yashgorana — APPROVED** [review]

**Pull request merged**